packages/broker/src/plugins/subscriber/SubscriberPlugin.ts

```diff
@@ -9,60 +10,39 @@ interface ConfigStream {
 
 export interface SubscriberPluginConfig {
     streams: ConfigStream[]
-    subscriptionRetryInterval: number
 }
 
 const logger = new Logger(module)
 
 export class SubscriberPlugin extends Plugin<SubscriberPluginConfig> {
-
-    private readonly streamParts: StreamPartID[]
-    private readonly subscriptionRetryInterval: number
-    private subscriptionIntervalRef: NodeJS.Timeout | null
+    private readonly streamParts: { id: string, partition: number }[]
+    private subscriptions: Subscription[] = []
 
     constructor(options: PluginOptions) {
         super(options)
         this.streamParts = this.pluginConfig.streams.map((stream) => {
-            return toStreamPartID(toStreamID(stream.streamId), stream.streamPartition)
+            const streamPart = toStreamPartID(toStreamID(stream.streamId), stream.streamPartition)
+            return {
+                id: StreamPartIDUtils.getStreamID(streamPart),
+                partition: StreamPartIDUtils.getStreamPartition(streamPart)
+            }
         })
-        this.subscriptionRetryInterval = this.pluginConfig.subscriptionRetryInterval
-        this.subscriptionIntervalRef = null
     }
 
     private async subscribeToStreamParts(): Promise<void> {
-        const node = await this.streamrClient!.getNode()
-        await Promise.all([
-            ...this.streamParts.map(async (streamPart) => {
-                node.subscribe(streamPart)
-            })
-        ])
-    }
-
-    private async subscriptionIntervalFn(): Promise<void> {
-        if (this.streamrClient) {
-            try {
-                await this.subscribeToStreamParts()
-            } catch (err) {
-                logger.warn(`Failed to (re-)subscribe (retrying in ${this.subscriptionRetryInterval / 1000} seconds)`, {
-                    err
-                })
-            }
-        }
-        this.subscriptionIntervalRef = setTimeout(() => this.subscriptionIntervalFn(), this.subscriptionRetryInterval)
+        this.subscriptions = await Promise.all(this.streamParts.map(({ id, partition }) => (
+            this.streamrClient.subscribe({ id, partition, raw: true })
+        )))
     }
 
     async start(): Promise<void> {
         await this.subscribeToStreamParts()
-        this.subscriptionIntervalRef = setTimeout(() => this.subscriptionIntervalFn(), this.subscriptionRetryInterval)
         logger.info('Started subscriber plugin')
```

Comment:
Could move this logging to the start of the method, and use the same wording as in the `stop` method (i.e. `Start subscriber plugin`).

Reply:
Not going to do in this PR. Reason: naming convention with rest of plugins.